modules/tests/programs/ncmpcpp/ncmpcpp.nix

```
@@ -0,0 +1,69 @@
+{...}: {
```

Is the `{...}:` needed? 

```suggestion
{
```